doc/modules/feature_extraction.rst

```diff
@@ -463,12 +463,12 @@ content of the documents::
       with 9 stored elements in Compressed Sparse ... format>
 
   >>> tfidf.toarray()                        # doctest: +ELLIPSIS
-  array([[ 0.81940995,  0.        ,  0.57320793],
+  array([[ 0.83205029,  0.        ,  0.5547002 ],
          [ 1.        ,  0.        ,  0.        ],
          [ 1.        ,  0.        ,  0.        ],
          [ 1.        ,  0.        ,  0.        ],
-         [ 0.47330339,  0.88089948,  0.        ],
-         [ 0.58149261,  0.        ,  0.81355169]])
+         [ 0.6       ,  0.8       ,  0.        ],
+         [ 0.6       ,  0.        ,  0.8       ]])
```

Comment:
The fact that the value of the computed array changed in examples means that backward compatibility is broken. This shouldn't happen.